pkg/vmimage/data/usr/local/bin/oc-get-all

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+set -x
+
+# check where we're running
+if [[ $HOST != master-* ]]
+then
+  echo "This geneva action can only be executed from one of the masters"
+  exit
+fi
+
+for target in group user deploy service pod route
+do
+  oc get --all-namespaces $target
```

Comment:
Does it make sense to return these in json or yaml by adding the output flag e.g `-o json`

Reply:
I thought about it, but since the output of the geneva action reencodes the output of the command in another json object it makes it kind of messy, wdyt?

Reply:
What is the current output example of this?

Reply:
Any particual reason we dont use golang kubeclient for this?

Reply:
for speed, we wanted to leverage the existing RunCommand action and not write up a whole new geneva action for this (but we will retrofit the get control plane pods into a more flexible geneva action)